src/act.informative.cpp

```diff
@@ -1713,16 +1713,16 @@ void look_at_room(struct char_data * ch, int ignore_brief, int is_quicklook)
       }
     } else {
       if (weather_info.sky >= SKY_RAINING) {
-          if(ch->in_room->vehicle == !VEH_BIKE) {
+          if(ch->in_veh && ch->in_veh->type == !VEH_BIKE) {
```

Comment:
Almost! This needs to be `!=` instead of `== !`. The former is 'not equal' and the latter is 'equal to not'. VEH_BIKE is defined as a number, and I don't know what the negation of a number is, but it'd probably be a bug.